backend/bitboxbase/handlers/handlers.go

```diff
@@ -49,13 +59,24 @@ func NewHandlers(
 ) *Handlers {
 	handlers := &Handlers{log: log.WithField("bitboxbase", "base")}
 
+	handleFunc("/status", handlers.getStatusHandler).Methods("GET")
+	handleFunc("/gethostname", handlers.getGetHostnameHandler).Methods("GET")
+	handleFunc("/channel-hash", handlers.getChannelHashHandler).Methods("GET")
+	handleFunc("/mountflashdrive", handlers.getMountFlashdriveHandler).Methods("GET")
```

Comment:
change relevant requests from GET to POST as discussed offline for any requests that modify anything on the Base